extensions/amp-analytics/0.1/variables.js

```diff
@@ -302,8 +302,19 @@ export class VariableService {
 
 /**
  * @param {!Window} win
+ */
+export function installVariableService(win) {
+  registerServiceBuilder(win,
+                         'amp-analytics-variables',
+                         VariableService,
+                         /* opt_factory */ undefined);
+}
+
+/**
+ * @param {!Window} win
  * @return {!VariableService}
  */
 export function variableServiceFor(win) {
-  return fromClass(win, 'amp-analytics-variables', VariableService);
+  installVariableService(win);
```

Comment:
Installing here may not be necessary if you add `installVariableService` calls to test-amp-analytics.js and test-variables.js.